deploy/operator.yaml

```diff
@@ -16,15 +16,16 @@ spec:
       containers:
         - name: app-metrics-operator
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/aerogear/app-metrics-operator:ag-9184
```

Comment:
This is pushed for convenience. The dev image is built and pushed for testing. I'll remove it before merging this one.